build/common.sh

```diff
@@ -34,6 +34,7 @@ readonly DOCKER_MACHINE_DRIVER=${DOCKER_MACHINE_DRIVER:-"virtualbox --virtualbox
 # This will canonicalize the path
 KUBE_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")"/.. && pwd -P)
 
+# shellcheck source=../hack/lib/init.sh
```

Comment:
I am torn on these.  We could have changed dir to KUBE_ROOT and used relative paths in the first place.  We did all this so moving files around did not hurt so much.

If we are going to essentially reify the root into relative paths in the scripts, maybe we should just do the more obvious thing?

Alternatively, could we just disable this check globally - it seems like more work than it is worth...

Reply:
Yes I agree!

I like the idea of using relative paths. It would make static checking using shellcheck easier and more effective plus we wouldn't be running subshells to see where we are in the (relatively straightforward) filesystem just so as to source neighboring scripts.

I would be happy to close this PR completely and make and piecemeal make changes to move towards using relative paths instead.

Reply:
> Alternatively, could we just disable this check globally - it seems like more work than it is worth...

This check is currently disabled globally (I did so in the first PR to set up the linter).

changing to KUBE_ROOT and using relative paths makes sense to me, might be a bit involved at this point though ...